src/databricks/labs/ucx/source_code/base.py

```diff
@@ -136,6 +137,35 @@ class Linter:
     def lint(self, code: str) -> Iterable[Advice]: ...
 
 
+class SQLLinter(Linter):
+
+    def lint(self, code: str) -> Iterable[Advice]:
+        try:
+            expressions = parse_sql(code, read='databricks')
+            for expression in expressions:
+                if not expression:
+                    continue
+                yield from self.lint_expression(expression)
```

Comment:
```suggestion
            for statement in parse_sql(code, read='databricks'):
                if not statement:
                    continue
                yield from self.lint_statement(statement)
```
it's a statement (higher level), not an expression (lower-level)